src/application/globals/overwriteCollectionSecurityValues.ts

```diff
@@ -1,15 +1,27 @@
 import { CollectionDefinition } from 'postman-collection'
-import { SecurityOverwrite } from 'types'
+import { AuthAttribute, SecurityOverwrite } from 'types'
 
 export const overwriteCollectionSecurityValues = (
   collectionJson: CollectionDefinition | Partial<CollectionDefinition>,
-  { apiKey, basic, bearer }: SecurityOverwrite
+  { apiKey, basic, bearer, awsv4, digest, edgegrid, ntlm, oauth1, oauth2 }: SecurityOverwrite
 ): CollectionDefinition => {
+  let defaultSec = false
+
   // Early exit if no auth is defined
-  if (!collectionJson.auth) return collectionJson
+  // if (!collectionJson.auth) return collectionJson
+
+  // Check default security types
+  if (apiKey !== undefined || basic !== undefined || bearer !== undefined) {
+    defaultSec = true
+  }
 
   // Handle OAS securitySchemes type apiKey
-  if (collectionJson?.auth?.apikey && Array.isArray(collectionJson.auth.apikey) && apiKey?.value) {
+  if (
+    collectionJson?.auth?.apikey &&
+    Array.isArray(collectionJson.auth.apikey) &&
```

Comment:
since you're checking with Array.isArray, you can just move to 

```
Array.isArray(collectionJson?.auth?.apikey)
```
and nuke the first check